config_tors.yml

```diff
@@ -1,11 +1,8 @@
 ---
-
 - name: config tors
   hosts: localhost
-  gather_facts: true
```

Comment:
This gathers variables (facts) about your target host, which in this case is localhost. Since you do not use any of these (system) variables, you can leave it out to speed things up.